### DISCUSSION TITLE
Prazo para inscrição

### DISCUSSION THREAD

**Discussion by viniciusfersil123** [Nexo-Tec/time#3] [Q&A] [answered]
Boa noite, gostaria de saber qual data limite para inscrição na vaga. Muito Obrigado !

**Comment by ibrahimcesar** [comment, marked as answer]
Olá @viniciusfersil123 boa noite! Tudo bem? Nós vamos ir marcando conversas após a submissão ao longo de Maio, mas entendemos que alguns dos desafios podem levar um tempo menor or maior.

Sendo bem honesto não definimos nenhuma data específica além de "maio" para fazer o processo até por que almejamos mais ter conversas de qualidade do que quantidade e até por ser bem experimental para nossa organização, não sabíamos exatamente como colocar prazos já que se trata menos de um grande funil e mais do que interesse de quem venha a querer se engajar. Mas eu diria que até 01/06 gastaríamos de ter preenchido senão todas todas, ao menos 2 de nossas vagas.

**Reply by viniciusfersil123** [reply]
Opa, muito obrigado pela presteza no retorno @ibrahimcesar e pela clara explicação da perspectiva do processo.

Abs
